ansible/roles/nginx/tasks/install.yml

```diff
@@ -38,3 +38,51 @@
 - name: Create the link for site enabled specific configurations
   sudo: yes
   file: path=/etc/nginx/sites-enabled/default state=link src=/etc/nginx/sites-available/default
+
+- name: Create directories for certificates
+  file:
+    path: "{{ item }}"
+    owner: "root"
+    group: "root"
```

Comment:
The certs are current owned by root, and apache is running as www-data (groups: www-data cchq). I think fine to leave these as root.